css/doc-explorer.css

```diff
@@ -0,0 +1,103 @@
+#graphiql-container .docExplorerWrap {
+  position: absolute;
+  top: 0;
+  right: 0;
+  height: 100%;
+  box-shadow: 0 0 8px rgba(0, 0, 0, 0.15);
+  z-index: 2;
+}
+
+#graphiql-container .doc-explorer {
+  height: 100%;
+}
+
+#graphiql-container .doc-explorer-title-bar {
+  position: relative;
+  height: 34px;
+  border-left: solid 1px #d6d6d6;
+  letter-spacing: 1px;
+  line-height: 14px;
+  padding: 7px 14px 6px;
+  -webkit-user-select: none;
+          user-select: none;
+}
+
+#graphiql-container .doc-explorer-title {
+  padding: 10px 0 10px 10px;
+  font-size: 14px;
+  text-align: right;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow-x: hidden;
+}
+
+#graphiql-container .doc-explorer-toggle-button,
+#graphiql-container .doc-back-to-main-button {
+  position: relative;
+  float: left;
+  padding: 1px 8px;
+  line-height: 22px;
+  background-color: #f6f7f8;
+  border-radius: 2px;
+  border: 1px solid;
+  border-color: #cccccc #c5c6c8 #b6b7b9;
+  font-weight: bold;
+  top: 50%;
+  -webkit-transform: translate(0, -50%);
+          transform: translate(0, -50%);
+}
+
+#graphiql-container .doc-explorer-contents {
+  position: relative;
+  height: 100%;
+  background-color: #ffffff;
+  border-left: 1px solid #d6d6d6;
+  border-top: 1px solid #d6d6d6;
+  padding: 14px 15px;
+  overflow: hidden;
```

Comment:
It seems that `overflow: auto` would be a better choice here. If you have a big schema it overflows the screen and can't be scrolled.